.github/workflows/check_cla_ruleset.yml

```diff
@@ -0,0 +1,17 @@
+# triggered on all repositories via rulesets
```

Comment:
Rulesets workflows don't have to be in `.github/workflows/` do they? Better to move them out then so they will be only triggered from rulesets, not as this repository workflows.

Reply:
It should run the same way that it is expected to run for any other repository: because it is enforced by organization-level ruleset that this repository belongs to. It shouldn't be required to have it running for this repository as repository workflows.

Reply:
I mean I agree it should also be so, but it does not seem like they allow it. This is the error I get when I try to add workflow not in `.github/workflows/` as a ruleset.

<img width="467" alt="Screenshot 2023-10-30 at 15 18 38" src="https://github.com/dfinity/public-workflows/assets/47304080/fa8d2079-6861-4798-90f4-c4ab1ed33118">